example/src/main/java/com/stripe/example/activity/SimpleRedirectPaymentMethodActivity.kt

```diff
@@ -0,0 +1,205 @@
+package com.stripe.example.activity
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.widget.ArrayAdapter
+import androidx.appcompat.app.AppCompatActivity
+import com.stripe.android.ApiResultCallback
+import com.stripe.android.PaymentConfiguration
+import com.stripe.android.PaymentIntentResult
+import com.stripe.android.Stripe
+import com.stripe.android.model.ConfirmPaymentIntentParams
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.model.PaymentMethodCreateParams
+import com.stripe.example.R
+import com.stripe.example.databinding.SimpleRedirectActivityBinding
+import com.stripe.example.module.BackendApiFactory
+import com.stripe.example.service.BackendApi
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.schedulers.Schedulers
+import okhttp3.ResponseBody
+import org.json.JSONException
+import org.json.JSONObject
+import java.io.IOException
+import java.lang.ref.WeakReference
+
+class SimpleRedirectPaymentMethodActivity : AppCompatActivity() {
+    private val stripe: Stripe by lazy {
+        Stripe(this, PaymentConfiguration.getInstance(this).publishableKey)
+    }
+    private val viewBinding: SimpleRedirectActivityBinding by lazy {
+        SimpleRedirectActivityBinding.inflate(layoutInflater)
+    }
+    private val keyboardController: KeyboardController by lazy {
+        KeyboardController(this)
+    }
+    private val compositeSubscription = CompositeDisposable()
+    private val backendApi: BackendApi by lazy {
+        BackendApiFactory(this).create()
+    }
```

Comment:
We do this in a lot of examples, I wonder if we should create a `StripeExampleActivity` superclass. Doesn't have to be in this PR.

Reply:
Yeah, I was wondering the same thing